### DISCUSSION TITLE
Google Cloud Monitoring (Stackdriver) open source plugin - Add PromQL support as a querying language

### DISCUSSION THREAD

**Discussion by dsaintilma-flinks** [grafana/grafana#64349] [Plugins]
Hi,

Since Google Cloud Monitoring plugin/datasoure now supports PromQL, it would great to have this querying language added as an option for those of use who prefers using that language.

Thanks!

**Comment by asimpson** [comment]
Hi @dsaintilma-flinks, thanks for the comment! I agree, it would be great if we had support for PromQL in Cloud Monitoring. This is something I will look into and hopefully prioritize with the team.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82763. Feel free to continue the discussion around this there. Thank you!
